README.md

```diff
@@ -63,6 +63,20 @@ coming soon
 
 # Working on Docs
 
+## Datadog Staff
+
+* Always branch off of master
+* Create a branch `slack_handle/your_feature` if you would like to create a preview site and run tests
```

Comment:
I don't think the 'if' is meant to operate on 'Create'; it operates on 'you would like to create a preview site" 

Probably 'Name' is better: "**Name** your branch `slack_handle/your_feature` if you would like to..."